From 936928b81f73340a7f04f1f043cfe33bd9fdd4bb Mon Sep 17 00:00:00 2001
From: Harsh J <harsh@cloudera.com>
Date: Mon, 26 Nov 2012 17:05:02 +0530
Subject: [PATCH 1216/1357] MR1: MAPREDUCE-3678. The Map tasks logs should carry the input split it processed.

Description: MR map tasks don't currently log the splits they use, so its harder to debug to find the file behind a specific failed task.
Reason: Customer Request
Author: Harsh J
Ref: CDH-6951
---
 src/mapred/org/apache/hadoop/mapred/MapTask.java |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/mapred/org/apache/hadoop/mapred/MapTask.java b/src/mapred/org/apache/hadoop/mapred/MapTask.java
index 17cb4e8..8898563 100644
--- a/src/mapred/org/apache/hadoop/mapred/MapTask.java
+++ b/src/mapred/org/apache/hadoop/mapred/MapTask.java
@@ -432,6 +432,7 @@ class MapTask extends Task {
       job.setLong("map.input.start", fileSplit.getStart());
       job.setLong("map.input.length", fileSplit.getLength());
     }
+    LOG.info("Processing split: " + inputSplit);
   }
 
   static class NewTrackingRecordReader<K,V> 
@@ -635,6 +636,7 @@ class MapTask extends Task {
     org.apache.hadoop.mapreduce.InputSplit split = null;
     split = getSplitDetails(new Path(splitIndex.getSplitLocation()),
         splitIndex.getStartOffset());
+    LOG.info("Processing split: " + split);
 
     org.apache.hadoop.mapreduce.RecordReader<INKEY,INVALUE> input =
       new NewTrackingRecordReader<INKEY,INVALUE>
-- 
1.7.0.4

